src/build.schema.json

```diff
@@ -37,6 +37,11 @@
           "type": "string"
         }
       },
+      "labels": {
+        "additionalProperties": {},
```

Comment:
Shouldn't this be validated to be a object of strings?
```suggestion
        "additionalProperties": {
          "type": "string"
        },
```

Reply:
Generally they should be strings, but I don't think that is required :man_shrugging: